src/microbe_stage/MicrobeWorldEnvironment.cs

```diff
@@ -18,6 +20,11 @@ public override void _ExitTree()
         Settings.Instance.BloomStrength.OnChanged -= OnStrengthChanged;
     }
 
+    public void UpdateAmbientReflection(Color colour)
+    {
+        ((ShaderMaterial)Environment.Sky.SkyMaterial).SetShaderParameter(skyColorShaderParameter, colour);
```

Comment:
This should have an optimization where if this is called with the same `colour` as before this wouldn't call the shader update.

Also the cast to `(ShaderMaterial)` should be done in _Ready / _EnterTree so that it doesn't happen each frame.